source/sass/wagtail/blocks/annotated-image-block.scss

```diff
@@ -0,0 +1,9 @@
+// .image-block {
+//   .wide {
+//     @extend .no-gutters;
+//     @extend .px-0;
+//     @extend .mx-0;
+//     @extend .w-100;
+//     @extend .mw-100;
+//   }
+// }
```

Comment:
If this file isn't used anymore, let's delete it! 💪

Reply:
Another great catch, deleted!